scripts/00spinup.py

```diff
@@ -57,23 +61,18 @@
 # Initialization
 # ------------------------------------------
 # Arrays for total values (across all layers), with time as the first axis
-subsurface_storage = np.zeros((nt,))
-surface_storage = np.zeros((nt,))
-wtd = np.zeros((nt, nx, ny))
-et = np.zeros((nt,))
-overland_flow = np.zeros((nt, nx, ny))
+subsurface_storage = np.zeros(nt)
+surface_storage = np.zeros(nt)
+wtd = np.zeros((nt, ny, nx))
+et = np.zeros(nt)
```

Comment:
I wasn't aware that for the 1D case, the shape can be specified as a scalar instead of a 1-tuple. I agree - this syntax is simpler.